### PR TITLE
More edits from the list.

### PR DESCRIPTION
**Pull request by chris-wood**

### CODE REVIEW

File: draft-ietf-tls-ticketrequests-02.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
+---
```
Comment (DavidSchinazi) [inline draft-ietf-tls-ticketrequests-02.md]: This file is not necessary, why not have a git tag when submitting?

File: draft-ietf-tls-ticketrequests.md [hunk 2]
```diff
@@ -130,19 +131,18 @@ struct {
...
 
-A supporting server MAY vend TicketRequestContents.count NewSessionTicket messages to a
-requesting client, and SHOULD NOT send more than TicketRequestContents.count NewSessionTicket
-messages to a requesting client. Servers SHOULD place a limit on the number of tickets they are willing to
-vend to clients. Thus, the number of NewSessionTicket messages sent should be the minimum of
-the server's self-imposed limit and TicketRequestContents.count. Servers MUST NOT send more
-than 255 tickets to clients.
+A supporting server MAY use TicketRequestContents.count when determining how many NewSessionTicket 
+messages to send to a requesting client, and SHOULD place a limit on the number of tickets sent.
+Typically, the number of NewSessionTicket messages sent should be the minimum of the server's
```
Comment (DavidSchinazi) [inline draft-ietf-tls-ticketrequests.md]: s/should/SHOULD/

File: draft-ietf-tls-ticketrequests.md [hunk 3]
```diff
@@ -159,7 +159,12 @@ and use tickets beyond common lifetime windows of, e.g., 24 hours. Despite ticke
 hints provided by servers, clients SHOULD dispose of pooled tickets after some reasonable
 amount of time that mimics the ticket rotation period.
 
+Servers that do not enforce a limit on the number of NewSessionTicket messages sent in response
+to a "ticket_request" extension may leave themselves open to DoS attacks, especially if ticket
```
Comment (DavidSchinazi) [inline draft-ietf-tls-ticketrequests.md]: s/may/could/

**Review by DavidSchinazi — APPROVED** [review]
Approved pending

File: draft-ietf-tls-ticketrequests-02.md (hunk 1 above)
Comment (tfpauly) [reply draft-ietf-tls-ticketrequests-02.md]: I think this was just a mistake

File: draft-ietf-tls-ticketrequests.md (hunk 3 above)
Comment (chris-wood) [reply draft-ietf-tls-ticketrequests.md]: Fixed.

File: draft-ietf-tls-ticketrequests.md (hunk 2 above)
Comment (chris-wood) [reply draft-ietf-tls-ticketrequests.md]: Fixed, and removed "Typically,"

File: draft-ietf-tls-ticketrequests-02.md (hunk 1 above)
Comment (chris-wood) [reply draft-ietf-tls-ticketrequests-02.md]: Yep, fat fingered.

File: draft-ietf-tls-ticketrequests.md [hunk 4]
```diff
@@ -90,27 +90,28 @@ be useful to use multiple, distinct tickets when opening parallel connections. C
...
 services to vend tickets to clients to use for accelerated handshakes with early data. (Note that
 if early data is not needed by these connections, this method SHOULD NOT be used. Fresh handshakes
 SHOULD be performed instead.)
 - Less ticket waste: Currently, TLS servers use application-specific, and often implementation-specific,
 logic to determine how many tickets to issue. By moving the burden of ticket count to clients,
-servers do not generate wasteful tickets for clients. Moreover, as ticket generation may involve
-expensive computation, e.g., public key cryptographic operations, avoiding waste is desirable.
-- Decline resumption: Clients may indicate they have no intention of resuming connections by
+servers do not generate wasteful tickets. As an example, clients might only request one ticket during
+resumption. Moreover, as ticket generation may involve expensive computation, e.g., public key
```
Comment (tfpauly) [inline draft-ietf-tls-ticketrequests.md]: Looks like we're removing `may` words, but we have one here. How about `might`?

```suggestion
resumption. Moreover, as ticket generation might involve expensive computation, e.g., public key 
```

**Review by tfpauly — APPROVED** [review]

File: draft-ietf-tls-ticketrequests.md (hunk 4 above)
Comment (chris-wood) [reply draft-ietf-tls-ticketrequests.md]: Works for me!

**Pull request merged**